### PR TITLE
Added Language orchestrator to CSharp and NodeJS.

### LINKED ISSUE #3737: Update Orchestrator with Custom Question Answering
### Use this [query](https://github.com/Microsoft/botbuilder-samples/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen++label%3Aenhancement) to search for the most popular feature requests.

**Is your feature request related to a problem? Please describe.**
Currently 14.nlp-with-orchestrator uses QnAMaker which is about to be deprecated. Sample should be updated with Custom Question Answering

**Describe the solution you'd like**
Update 14.nlp-with-orchestrator with Custom Question Answering.

**Additional context**
We already have samples 12.CustomQABot and 48.CustomQABotAllFeatures that demonstrate the capabilities of Custom Question Answering.

### PR DESCRIPTION
**Pull request by Arsh-Kashyap**
Fixes #3737 

## Proposed Changes
Updated C# and NodeJS 14.nlp-with-orchestrator sample to call Custom Question Answering.

### CODE REVIEW

**Review by gabog — CHANGES_REQUESTED** [review]
Please remove changes made to sample 49 and make them as part of another issue and PR.

Thanks

File: samples/csharp_dotnetcore/49.qnamaker-all-features/Dialog/QnAMakerBaseDialog.cs [hunk 1]
```diff
@@ -22,7 +22,7 @@ public class QnAMakerBaseDialog : QnAMakerDialog
         public const string DefaultCardNoMatchResponse = "Thanks for the feedback.";
 
         private readonly IBotServices _services;
-        private readonly string DefaultAnswer = "No QnAMaker answers found.";
+        private readonly string DefaultAnswer = "";
```
[the remark this answers is not in the document]
Comment (Arsh-Kashyap) [reply samples/csharp_dotnetcore/49.qnamaker-all-features/Dialog/QnAMakerBaseDialog.cs]: I created a PR linked with a new issue [here](https://github.com/microsoft/BotBuilder-Samples/pull/3742)

**Comment by Arsh-Kashyap** [comment]
@gabog Please review this PR.

**Comment by tracyboehrer** [comment]
Closing without merge.  No longer needed.

**Comment by jrpavoncello** [comment]
@tracyboehrer Would you mind elaborating why this was no longer needed? I am soon going to be using the samples to help migrate our existing production bot project, and I didn't find any existing examples that tied all of Orchestrator, Custom Question Answering, and LUIS together. It seems like this might be helpful to bridge that gap.

**Pull request closed without merging**